gsgmail_process_email.py

```diff
@@ -654,7 +653,6 @@ def _handle_mail_object(self, mail, email_id, rfc822_email, tmp_dir, start_time_
 
                 self._parse_email_headers(parsed_mail, part, add_email_id=add_email_id)
 
-                self._base_connector.debug_print("part: {0}".format(part.__dict__))
                 self._base_connector.debug_print("part type", type(part))
```

Comment:
Do you have an example of what this would look like in reality? Wondering if we can remove it, too

Reply:
Trying to run the connector locally now and get a full example for you.

Emails consist of multiple "parts" depending on what components they have - a plain text part, an HTML part, and any number of attachment parts.

This line would print each of those in their entirety, leaking mail contents to the debug log. We should remove it. The "type" on the next line is still safe to print.